src/base/utils/string.h

```diff
@@ -49,6 +47,14 @@ namespace Utils
         bool naturalCompareCaseInsensitive(const QString &left, const QString &right);
 
         QString wildcardToRegex(const QString &pattern);
+
+        template <typename T>
+        T unquote(const T &str)
+        {
+            if (str.startsWith('\"') && str.endsWith('\"'))
```

Comment:
This function would fail with string `"\""`. You may want to copy implementation from `src/app/cmdlineoption.cpp` and use this new function there.